Bot/Distribution.py

```diff
@@ -35,6 +35,11 @@ async def pool_task(row, db):
                     )
                 elif missed_days < 3:
                     await Bot.bot.send_message(chat_id=telegram_id, text=Localize.DidNotFollowed[user_locale])
+                    str_to_send = ""
+                    # make a list of people to follow
+                    for i in range(len(to_follow_list)):
+                        str_to_send += f"{i}. https://github.com/{to_follow_list[i]}\n"
```

Comment:
NFS inline

```
lines = []
for idx, u in enumerate(to_follow_list):
  lines.append(f'{idx}. https://github.com/{u}')
str_to_send = '\n'.join(lines)
```
??

Reply:
`str_to_send = '\n'.join( [ f'{idx}. https://github.com/{u}' for idx, u in enumerate(to_fllow_list) ] )`..

Reply:
up to u!

Reply:
nah, we keep it user friendly